fedmsg_meta_fedora_infrastructure/scm.py

```diff
@@ -56,15 +57,17 @@ def long_form(self, msg, **config):
             except KeyError:
                 repo = '.'.join(msg['topic'].split('.')[5:-1])
 
+            namespace = msg['msg']['commit'].get('namespace', 'rpms')
             rev = msg['msg']['commit']['rev']
 
             seen = msg['msg']['commit'].get('seen', False)
             if seen:
                 return already_seen_msg
 
-            url = 'https://src.fedoraproject.org/cgit/' + \
-                '{repo}.git/patch/?id={rev}'
-            response = requests.get(url.format(repo=repo, rev=rev))
+            url = 'https://src.fedoraproject.org/' + \
+                '{namespace}/{repo}/c/{rev}.patch'
+            response = requests.get(url.format(
```

Comment:
Can you add a timeout to this? ``requests.get(..., timeout=30)`` or something. 

You'll want to catch all requests exceptions here as well since this API doesn't include raising any requests exceptions. ``requests.exceptions.RequestException`` is the general requests exception. I guess you could also call ``response.raise_for_status()`` to handle non-"okay" responses and then in the except block return a reasonable default explaining why the patch isn't included, otherwise return response.text.